certified-connectors/Leadinfo/readme.md

```diff
@@ -0,0 +1,32 @@
+# Leadinfo Certified Connector
```

Comment:
@GerbenZ01 Please follow and include all required section in the readme file. Please follow our template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md), and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).